lib/tree-view.coffee

```diff
@@ -117,6 +121,10 @@ class TreeView extends View
      'tree-view:toggle-vcs-ignored-files': -> toggleConfig 'tree-view.hideVcsIgnoredFiles'
      'tree-view:toggle-ignored-names': -> toggleConfig 'tree-view.hideIgnoredNames'
 
+    [0..8].forEach (index) =>
+      atom.commands.add @element, "tree-view:open-selected-entry-pane-#{index + 1}", =>
```

Comment:
What do you think about making this `tree-view:open-selected-entry-in-pane-#{index + 1}` just so it reads a bit more naturally when in the command palette?